openfisca_parsers/json_graph.py

```diff
@@ -70,16 +71,9 @@ def visit(ofnode):
         id_by_ofnode_address[ofnode_address] = ofnode_id
         jgfnode = merge(ofnode, {
             'id': ofnode_id,
-            'label': u'{}\n{}'.format(
-                ofnode['type'],
-                '\n'.join(map(
-                    lambda item: u'{} = {}'.format(*item),
-                    valfilter(lambda val: not isinstance(val, (dict, list)), dissoc(ofnode, 'type')).items(),
-                    )),
-                ).strip()
             })
         for key, value in ofnode.items():
-            if isinstance(value, list):
+            if key == 'operands':
```

Comment:
Perhaps exclude `path` key of `Parameter` node, rather than include *only* `operands`